appveyor.yml

```diff
@@ -92,16 +92,16 @@ for:
   # scripts to run after tests
   after_test:
   - ps: |
+      & pwd
       & Setup\BuildInstallers.cmd
-      if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
```

Comment:
❓ Why are these removed?

Reply:
I'm getting real tired of you not reading what I post as comments and commit messages.

Reply:
> Oh by the way.  Diagnostics will be removed once the pull request is approved.  I will rebase on top of master and cleanup as needed.  

https://github.com/gitextensions/gitextensions/commit/fc431a354490d6402cd445308c866f1f77585ab6

Reply:
@vbjay that's not nice to say. When reviewing everything as a whole and not each commit separately this info gets lost. Either way. Being nice and explaining to people what you think is "obvious" is part of the process. An important part which you shouldn't skip.

Reply:
Also, with your clarification, it's not at all obvious that removing this line was part of "diagnostics". Before posting something try to think how'd you react to a line like this had it no been written by you with all the context you have of the matter. It would probably look like a mistake.

Reply:
![screenshot_20190108-083020](https://user-images.githubusercontent.com/3629489/50833648-e8abd500-131f-11e9-88cb-5924c48d4647.png)

When reviewing I also read this before jumping on the changes.

Reply:
@vbjay I read the commit message for the change before asking the question. It says this:

> Temp commit to verify installer build
> list files

I did not understand the change or the commit message, so I asked the question.

Reply:
The commit is labeled Diagnostics.  I removed those lines trying to still get artifacts like tree output even if something failed.  It was a tool to see why the azure devops plugin when sourced from main release folder was not found but portable found it.  I finally just gave up on trying to source that file from release folder.   May revisit later.  It works so not such a big deal.  I was trying to source as much as I could from gitextensions release folder instead of build folders of each plugin project.  This will alert us to issues of files in release folder not matching plugin project output REAL quick.